gnome-image-installer/pages/install/gis-install-page.c

```diff
@@ -619,7 +617,7 @@ gis_install_page_verify (GisPage *page)
   const gchar *signature_path = gis_store_get_image_signature ();
   GFile *signature = g_file_new_for_path (signature_path);
   gint outfd;
-  gchar *args[] = { "gpg",
+  const gchar *args[] = { "gpg",
```

Comment:
This could actually be `const gchar * const args[]`, I think? Because both the array and each value are `const`.

Disclaimer: I haven’t tried this, and my knowledge of C array-vs-`const` semantics is poor at best.